starters/graphql-serverless-contentful/src/schema/post/index.ts

```diff
@@ -0,0 +1,2 @@
+export { postTypeDefs } from './post.typedefs';
+export { postResolvers } from './post.resolvers';
```

Comment:
I think we should add a test file for this post example since we added tests for the hello example.